crates/common/tedge_config/src/tedge_config_cli/tedge_config_defaults.rs

```diff
@@ -14,6 +14,9 @@ pub const DEFAULT_LOG_PATH: &str = "/var/log";
 pub const DEFAULT_RUN_PATH: &str = "/run";
 const DEFAULT_DEVICE_TYPE: &str = "thin-edge.io";
 
+pub const FILE_TRANSFER_ROOT_PATH_TEST: &str = "/tmp";
```

Comment:
As this is used only in tests, it's better to have it defined only in those test modules rather than here. There could be some other test that would want the `FILE_TRANSFER_ROOT_PATH` to be in some other directory (e.g: `/tmp/tedge/file-transfer/`). We don't need to corrupt a core module with test artefacts.